### DISCUSSION TITLE
Welcome to Discussions / 欢迎来到讨论区 !

### DISCUSSION THREAD

**Discussion by vthinkxie** [NG-ZORRO/ng-zorro-antd#5823] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

----------------

## 👋欢迎来到讨论区
   我们正在使用“讨论区”作为与社区其他成员联系的地方。 

   你可以在这里：
   * 问您想知道的问题。
   * 分享意见。
   * 与其他社区成员互动。

**Comment by Eve-Sama** [comment]
Nice!

**Reply by withOutUndo** [reply]
Nice!

**Comment by fxcvv** [comment]
v10正式版啥时候会上呀？

**Reply by vthinkxie** [reply]
9月份会发布
